serviceUsage.html

```diff
@@ -345,8 +362,94 @@ <h2>
+			which will make sure it is linked to the correct events. The lookups are used in filling the information for links, which will help users to know what event they want to query with the parameters so that they are automatically fetched and linked by Generate instead of the users adding them as a mandatory param when generating events.
+		</p>
+		<p> <em>parameters</em> specified should be enough to replace the listed event id as input will help to prepare the Query to ER and Lookup the ER data.</p>
+		<p><h6>Example:</h6>
+			As a User, I want to place the LINK type ARTIFACT to Event ID of EiffelArtifactCreatedEvent from Event Repository.
+			The user passes the below template information to REMReM Generate, REMReM Generate will Query to configured Event Repository and place the event ID in LINK's and generate the Eiffel message.
+		</p>
+		<h5> Templates for LOOKUPS : </h5>
+		<pre>
+		"links":[
+					{
+			"type": "ARTIFACT",
+			"%lookup%": {
+			"eventType": "EiffelArtifactCreatedEvent",
+			"properties": [
+				{ “data.identifier": "some purl identifier defining the artifact" }]
+					}
+				}]
+		</pre>
+		<p><h6>ER Query will look like:</h6></p>
+		<pre>https://host:port/eventrepository/events/?eventType=EiffelArtifactCreatedEvent&data.identifier=some purl identifier defining the artifact</pre>
+		<h5> Examples for Lookups:</h5>
+		<h6> Example 1:</h6>
+		<pre>
+		"links": [
+			"type": “ARTIFACT",​
+			"%lookup%": {​
+			"eventType": "EiffelArtifactCreatedEvent",​
+			"properties": [
+					{​“data.identity": "pkg:maven/com.ericsson.eiffel/testartifact@0.0.21?type=xml&classifier=test"}
+					]​
+			}​
+		]
+		</pre>
+		<h6> Example 2 : </h6>
+		<pre>
+		"links": [
+			"type": “ARTIFACT",​
+			"%lookup%": {​
+			"eventType": "EiffelArtifactCreatedEvent",​
+			"properties": [
+					{​“data.identity": "pkg:maven/org.apache.xmlgraphics/batik-anim@1.9.1?repository_url=repo.spring.io/release"}
+					]​
+			}​
+		]
+		</pre>
+		<h6> Example 3 (For SourceChangeCreated/SourceChangeSubmitted Events Identifier will be repoUri): </h6>
+		<pre>
+			"links": [
+			"type": “CHANGE",​
+			"%lookup%": {​
+			"eventType": "EiffelSourceChangeCreatedEventEvent",​
+			"properties": [
+					{​"data.gitIdentifier.repoUri": "https://gerrit.ericsson.se"​}
+					]​
+			}​
+		]
+		</pre>
+		<h5>Lookups are provided with two options in swagger page:</h5>
+		<table class="lookups">
+		<tr>
+            <th style="text-align: left;">Options</th>
+			<th style="text-align: left;">Default Value</th>
+            <th style="text-align: left;">Description</th>
+         </tr>
+		<tr>
+            <td><h6>failIfMultipleFound:</h6></td>
+			<td>False</td>
+            <td>If value is set to false then all upstream/downstream event ids of that particular event are fetched from event repository  and adds them in links section of the target field.
```

Comment:
Default value is false. This should also be mentioned

Reply:
I have kept default values in table format.